lib/remi/data_subjects/sftp_file.rb

```diff
@@ -1,4 +1,44 @@
 module Remi
+  module DataSubject::SftpFile
+
+    attr_reader :sftp_session
+
+    def sftp_retry(&block)
+      tries ||= 3
+
+      block.call
+    rescue RuntimeError => err
+      if (tries -= 1) > 0
+        logger.error "Error: #{err.message}"
+        logger.error "Will retry #{tries} more times"
+        sleep(1)
+        retry
+      else
+        raise err
+      end
+    end
+
+    def begin_connection
+      sftp_retry do
+        Timeout.timeout(30) do
```

Comment:
Yeah, I thought about that, but then I couldn't come up with a convincing use case for when I'd ever want to increase it.  

But you know, you're right....